app/views/carl_people/_header_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<<%= top ? "header" : "footer" %>>
+  <nav id="<%= top ? "top" : "bottom" %>">
+    <% unless top %>
+      <a href="#top" title="Jump to the top of the page!">Top</a>
+    <% end %>
+    <a href="/" title="Carl's Taskmaster">Home</a>
+    <!-- <a href="/reverse" title="Carl's Taskmaster">(newest task first)</a> -->
+    <!-- this isn't working for individual person views. come back to it later after finishing requirements. -->
+    <a href="/about" title="Learn more about Carl.">About</a>
+    <a href="/people" title="Who exists in this world?">People</a>
+    <a href="/create" title="Create a new task!">Create New Task</a>
+  </nav>
+</<%= top ? "header" : "footer" %>>
```

Comment:
I've implemented this as you suggested. I'm still pretty fuzzy about what logic belongs in .erb files. For example, I think my create/edit form (`app/views/carl_tasks/create.html.erb`) has a ton of logic to differentiate between the two processes. Is that something I should try to improve on, too?